src/commands/switch.rs

```diff
@@ -0,0 +1,187 @@
+use super::super::errors;
+use super::*;
+use crate::core::Target;
+use crate::tasks::*;
+use crate::{core, git};
+use clap::{App, Arg};
+
+pub struct SwitchCommand {}
+
+impl Command for SwitchCommand {
+    fn name(&self) -> String {
+        String::from("switch")
+    }
+
+    fn app<'a>(&self) -> App<'a> {
+        App::new(self.name().as_str())
+            .version("1.0")
+            .alias("s")
+            .about("switches to the specified branch within the current repository")
+            .after_help(
```

Comment:
Would you mind changing this to the `.long_about` type? (`clap-3.0.0.beta.2` switched this out).
```suggestion
            .long_about(
```

Reply:
Addressed this in latest PR!